### ISSUE TITLE
Bug: Unable to edit multi-day resource reservation

### ISSUE THREAD

**Issue by dinosound** [LibreBooking/app#81]
When a reservation is made using the "repeat" option, no edits/deletions can be made after the fact - either to the original date or the repeat dates.

**Comment by effgarces** [comment]
I,ve only used the repeat option once or twice in an older version, but seem to recall, that it was possible to edit/delete. I've marked as a bug, but still need to test and reproduce. If you have further info on it, error logs, etc. it would be helpfull

**Comment by scmu49** [comment]
I have same problem, here the log 
Tue Oct 11 16:57:00.539286 2022] [php7:warn] [pid 6840:tid 1780] [client 10.10.30.1:7963] PHP Warning:  count(): Parameter must be an array or an object that implements Countable in C:\\xampp\\htdocs\\Booked\\tpl_c\\b78fcaf096b73254cead3e799494561e4c5bd2c3_0.file.pdf.tpl.php on line 167, referer: http://xxx/booked/Web/schedule.php

How to solve?

**Comment by effgarces** [comment]
Thanks for the log, I will try to reproduce this on my test machine and see if I can pinpoint what is causing it

**Comment by scmu49** [comment]
Thank you very much, i'm looking forward to your update

**Comment by Andy-xy** [comment]
Found a solution:
in **...\xampp\htdocs\booked\tpl\Reservation\pdf.tpl** I edited row 120:

old: {if count($RepeatWeekdays) gt 0}
new: {if (is_array($RepeatWeekdays) && count($RepeatWeekdays) gt 0)}

Now it's working for me...

**Comment by LpSolit** [comment]
> @effgarces : same code has to be edited in **\xampp\htdocs\booked\tpl\Reservation\view.tpl** at row 66.

Yes, I confirm that. Else you get an "unknown error" message instead of being able to edit the reservation. Trivial fix, please commit.

**Comment by JohnVillalovos** [comment]
Issue has been resolved.
